src/sentry/static/sentry/app/views/performance/compare/spanDetailContent.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import map from 'lodash/map';
+
+import {t} from 'app/locale';
+import getDynamicText from 'app/utils/getDynamicText';
+import DateTime from 'app/components/dateTime';
+import {SpanType, rawSpanKeys} from 'app/components/events/interfaces/spans/types';
+import {SpanDetails, Row, Tags} from 'app/components/events/interfaces/spans/spanDetail';
+
+type Props = {
+  span: Readonly<SpanType>;
+};
+
+class SpanDetailContent extends React.Component<Props> {
+  render() {
```

Comment:
This could be a converted into a function component in a follow up pull request.

Reply:
should be addressed in https://github.com/getsentry/sentry/pull/18610/commits/969a783ebb2dc947317e040c22e70622b35fa2b7